templates/_navbar.html.twig

```diff
@@ -34,8 +34,7 @@
 							{% else %}
 								<ul class="register">
 									<li class="nav-item">
-										<a href="{{ path('login') }}">
-											Inscription / Connexion</a>
+										<a href="/register">Inscription</a>&nbsp;<a href="{{ path('login') }}">/ Connexion</a>
```

Comment:
Use a path.

Reply:
![8dj47q](https://github.com/WildCodeSchool/2023-09-PHP-LYON-P3-eclat-d-art/assets/109858344/eda550ae-ba2a-4f52-b7c1-a69e9b8f89dc)

Reply:
Where are the changes??